dockerfile/best-practice/missing-zypper-clean.yaml

```diff
@@ -1,11 +1,11 @@
 rules:
   - id: missing-zypper-clean
     severity: WARNING
-    languages: [generic]
+    languages: [dockerfile]
     patterns:
-      - pattern: zypper $COMMAND
-      - pattern-not-inside: RUN ... && zypper clean
-      - pattern-not-inside: RUN ... && \ zypper clean
+      - pattern: RUN ... zypper ...
+      - pattern-not: RUN ... zypper clean
```

Comment:
We removed enforcing the `&&` condition here - intentional?
What happens if it's e.g `|| zypper clean`? Is this OK?